chezbetty/models/user.py

```diff
@@ -117,7 +117,7 @@ class User(account.Account):
     _salt     = Column("salt", String(255))
     enabled   = Column(Boolean, nullable=False, default=True)
     archived  = Column(Boolean, nullable=False, default=False)
-    role      = Column(Enum("user", "serviceaccount", "manager", "administrator", name="user_type"),
+    role      = Column(Enum("user", "volunteer", "serviceaccount", "manager", "administrator", name="user_type"),
```

Comment:
This needs to be added to the end of the list of options, not in the middle because SQLAlchemy will use a native ENUM type not a String for many engines. There may also need to be a migration written to add the type to the enum definition.

Reply:
We use postgres, right?  The migration I wrote inserts the new enum value into the correct place, so "volunteer" shouldn't need to be added to the end of the list...but I can do that if it's an issue for other engines.

Reply:
I think if this works for postgres that is good enough.